pytest-wdl-tests/tests/expr/types/primitive_types/test_primitive_types.wdl

```diff
@@ -0,0 +1,10 @@
+version 1.0
+
+workflow test_types_wf {
+    Int i = 0                               # An integer value
+    Float float = 27.3                      # A floating point number
+    Boolean bool = true                     # A boolean true
+    Boolean bool2 = false                   # A boolean false
+    String str = "hello, $!~ \n\t\r world"  # A string value
```

Comment:
IIRC there were a couple of other control characters singled out. Do we want to add tests for any of the others?

> An escape sequence starting with \\, followed by one of the following characters: \\, ", ', [nrbtfav], ?